benchmarks/kernels/benchmark_moe.py

```diff
@@ -31,18 +31,35 @@ def benchmark_config(
     topk: int,
     dtype: torch.dtype,
     use_fp8: bool,
+    use_int8: bool,
```

Comment:
`use_fp8` benchmarks FP8 with W8A8 while your current implementation of INT8 is just W8A16. Given this I would prefer to be more explicit here so could you please change them to `use_fp8_w8a8` and `use_int8_w8a16` respectively?

Reply:
Addressed